TestKickStart.bat

```diff
@@ -0,0 +1,8 @@
+
+@ECHO ON
+
+ECHO Python 2.7 Tests ===============================
+py -2.7 TestRunner.py
```

Comment:
Seems like a good approach to me, I didn't even know you had these virtual environments available by default.